Sources/StreamChatUI/ChatThread/ChatThreadVC.swift

```diff
@@ -276,7 +276,12 @@ open class ChatThreadVC: _ViewController,
         footerViewForMessage message: ChatMessage,
         at indexPath: IndexPath
     ) -> ChatMessageDecorationView? {
-        nil
+        guard components.threadRepliesCounterEnabled, message == messages.last else {
+            return nil
+        }
+        let repliesCounterDecorationView = components.threadRepliesCounterDecorationView.init()
+        repliesCounterDecorationView.content = L10n.Message.Thread.Replies.count(message.replyCount)
```

Comment:
I know this is following the current pattern, but maybe this is something we should be discussing at some point. Not for this PR, though.

Reply:
What part are you referring to? The Localisation?

Reply:
Nope, the way we are passing data to the views. IMO this view should just get a model that contains the replies count. The responsibility of formatting the data should be in the (view) model of the view itself.